capacity_by_user.py

```diff
@@ -251,10 +255,10 @@ def to_dollars(adjust):
 
 def process_command_line(args):
     parser = ArgumentParser()
-    parser.add_argument("-U", "--user", default="admin",
+    parser.add_argument("-U", "--user", required=True,
             help="The user to connect as (default: %(default)s)")
 
-    parser.add_argument("-P", "--password", default="admin",
+    parser.add_argument("-P", "--password",
         help="The password to connect with (default: %(default)s)")
```

Comment:
The "default" string doesn't make a lot of sense now
```suggestion
    parser.add_argument("-P", "--password",
        help="The password for the specified user. If not specified, you will be prompted later.")
```

Reply:
Yep, good catch.